framework/src/main/java/org/apache/drill/test/framework/Utils.java

```diff
@@ -169,7 +170,21 @@ public static ConnectionPool getConnectionPool() {
         boolean foundTests = false;
         for (String testGroup : testGroups) {
           if (categories != null && !categories.contains(testGroup)) {
-            continue;
+            if (categories.contains("smoke") &&
```

Comment:
The reason "smoke" is used as a category here is to show another way to incorporate the Simba drivers.  The one that we agreed upon is to add the Simba categories to the json files.  But this requires constant maintenance as we add more json files.  At the time, it was preferred not to make more changes to the framework than necessary.

So I decided to prototype what change would be needed to automatically invoke the Simba categories without changing the json files, to better evaluate the complexity.  In this case, I'm using "smoke" as a reference.  If a json file uses the "smoke" category, the framework will automatically add the Simba category.

We can go either way.  I have made all the changes to the json files, which will be in a separate PR.